index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+    <title>My Portfolio</title>
+</head>
+<body>
+        <!-- header and navbar -->
+        <header  class="navbar">
+            <nav>
+                <img src="images/App Bar - Compact - Dark - iOS.png" alt="" srcset="">
+            </nav>

```

Comment:
- [ ] kindly consider adding the contents of this header  as separate not adding it as one picture to follow the below 👇 requirements
> 1. The logo is on the left and has a link (`a` tag).
   * This is for three reasons: convention, semantics, and user experience(UX).
  > 2. The hamburger button is on the right and has a `nav` tag.
   * The `nav` tag is for semantics reasons and the right position for convention and user experience for mobile devices.

> 3. The content is aligned with Flexbox.
   * This gives us a lot of options to align all our elements, plus, it's responsive.
  
doing so will make your header meet all the requirements